modules/op-configuration-options-with-default-values.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: REFERENCE
+[id="op-configuration-options-with-default-values_{context}"]
+= Configuration options with default values
+
+[role="_abstract"]
+The following is a list of modifiable fields with default values in the `TektonConfig` CR.
+
+* `running-in-environment-with-injected-sidecars` (default: `true`): Set this option to `false` if pipelines are running in a cluster that do not use injected sidecars, such as Istio. Setting it to `false` decreases the time it takes for a task run to start.
++
+NOTE: For clusters that use injected sidecars, setting this option to `false` can lead to unexpected behavior.
+
+* `await-sidecar-readiness` (default: `true`): Set this option to `false` to stop {pipelines-shortname} from waiting for `TaskRun` sidecar containers to be running before it begins to operate. This allows tasks to be run in environments that do not support the `downwardAPI` volume type.
++
+NOTE: For clusters that use injected sidecars, setting this option to `false` can lead to unexpected behavior.
+
+* `default-service-account` (default: `pipeline`): This option contains the default service account name to use for `TaskRun` and `PipelineRun`, if none is specified.
+
+* `require-git-ssh-secret-known-hosts` (default: `false`): Setting this option to `true` requires that any Git SSH secret must include the `known_hosts` field. 
+
+** See _Configuring SSH authentication for Git_ in the _Additional resources_ section for more information about Git SSH secrets.
+
+* `enable-tekton-oci-bundles` (default: `false`): Set this option to `true` to enable the use of an experimental alpha feature, Tekton OCI bundle.
+
+* `embedded-status` (default: `both`): This option has three acceptable values:
+
+** `full`: Enables full embedding of `Run` and `TaskRun` statuses in the `PipelineRun` status.
+
+** `minimal`: Populates the `ChildReferences` field in the pipeline run status with name, kind, and API version information for each run and task run.
+
+** `both`: Applies both, `full` and `minimal` value.
++
+[NOTE]
+====
+The `embedded-status` flag is deprecated and will be removed in {pipelines-shortname} 1.11. In addition, the pipeline default embedded status will be changed to `minimal`.
+====
+
+* `enable-api-fields` (default: `stable`): Setting this option determines which gated features are enabled. Acceptable values are `stable`, `beta`, or `alpha`.
++
+NOTE: The `alpha` option is not supported.
```

Comment:
@shipsing I was advised by QE and SME to add the note so that the user knows that with `alpha` they lose support, but it is still an acceptable feature. 
It is discussed in this thread: https://github.com/openshift/openshift-docs/pull/60075/files#r1198969071

So, in that case, do you think it should be mentioned, or should not be mentioned at all?

Reply:
That sounds good. Thank you!